presto-docs/src/main/sphinx/gcp/storage.rst

```diff
@@ -0,0 +1,216 @@
+    </property>
+    <property>
+        <name>fs.AbstractFileSystem.gs.impl</name>
+        <value>com.google.cloud.hadoop.fs.gcs.GoogleHadoopFS</value>
+        <description>
+            The AbstractFileSystem for gs: (GCS) uris. Only necessary for use with Hadoop 2.
+        </description>
+    </property>
+    <property>
+        <name>fs.gs.project.id</name>
+        <value></value>
+        <description>
+            Required. Google Cloud Project ID with access to configured GCS buckets.
+        </description>
+    </property>
+    <!--
+    The following property is required when Presto is NOT running in a GCE VM.
+    The connector will look to see if it is running on a GCE VM with some level of GCS access in it's service account scope, and use that service account.
+    -->
+    <property>
+        <name>fs.gs.auth.service.account.json.keyfile</name>
+        <value></value>
+        <description>
+            The path to the json keyfile for the service account.
+        </description>
+    </property>
+    <!--
+    The following property is required only when Presto must run as different user than Hive metastore and Hive metastore uses StorageBasedAuthorization.
+    The connector presents files with '700' POSIX permissions by default, and the above combination causes Hive metastore to deny access to Presto.
+    -->
+    <property>
+        <name>fs.gs.reported.permissions​</name>
+        <value>777</value>
+        <description>
+          Optional. Override the default '700' simulated POSIX file permissions.
+        </description>
+    </property>
+
+More configuration properties and their default values are shown in `gcs-core-default.xml <https://github.com/GoogleCloudPlatform/bigdata-interop/blob/v1.9.10/gcs/conf/gcs-core-default.xml>`_.
+
+Hive Connector
+^^^^^^^^^^^^^^
+
+Another requirement is that you have enabled and configured a Hive connector in Presto.
+The connector uses Hive metastore for data discovery and is not limited to data residing on HDFS.
+
+**Configuring Hive Connector**
+
+* URL to Hive metastore:
+
+    * New Hive metastore on GCP:
+
+        If your Presto nodes are provisioned by GCP, your Hive metastore should also be on GCP to minimize latency and costs.
+        The simplest way to create a new Hive metastore on GCP is to create a small Cloud DataProc cluster (1 master, 0 workers), accessible from your Presto cluster.
+        Follow the steps for existing Hive metastore after finishing this step.
+
+    * Existing Hive metastore:
+
+        To use an existing Hive metastore with a Presto cluster, you need to set the ``hive.metastore.uri`` property in your Hive catalog properties file to ``thrift://${METASTORE_ADDRESS}:${METASTORE_THRIFT_PORT}``.
+        If the metastore uses authentication, please refer to `Hive Security Configuration`_.
+
+        .. _Hive Security Configuration: https://docs.starburstdata.com/latest/connector/hive-security.html
```

Comment:
Use a relative link. Also, since the document title has the link text we want, it can be inline:
```
If the metastore uses authentication, please refer to :doc:`hive-security`.